### ISSUE TITLE
vagrant up failed, npm package installation failed

### ISSUE THREAD

**Issue by wazum** [mblaschke/vagrant-development#84]
a simple `vagrant up --provider=virtualbox` fails (multiple times) with the following errors:

```
==> default: failed: [localhost] => (item=phantomjs) => {"cmd": "/usr/bin/npm install --global --production phantomjs", "failed": true, "item": "phantomjs", "rc": 1}
==> default: stderr: npm http GET https://registry.npmjs.org/phantomjs
==> default: npm http 304 https://registry.npmjs.org/phantomjs
==> default: npm WARN deprecated phantomjs@2.1.7: Package renamed to phantomjs-prebuilt. Please update 'phantomjs' package references to 'phantomjs-prebuilt'
==> default: npm http GET https://registry.npmjs.org/extract-zip
==> default: npm http GET https://registry.npmjs.org/fs-extra
==> default: npm http GET https://registry.npmjs.org/hasha
==> default: npm http GET https://registry.npmjs.org/progress
==> default: npm http GET https://registry.npmjs.org/kew
==> default: npm http GET https://registry.npmjs.org/request
==> default: npm http GET https://registry.npmjs.org/request-progress
==> default: npm http GET https://registry.npmjs.org/which
==> default: npm http 304 https://registry.npmjs.org/fs-extra
==> default: npm http 304 https://registry.npmjs.org/progress
==> default: npm http 304 https://registry.npmjs.org/kew
==> default: npm http 304 https://registry.npmjs.org/request
==> default: npm http 304 https://registry.npmjs.org/request-progress
==> default: npm http 304 https://registry.npmjs.org/which
==> default: npm http 304 https://registry.npmjs.org/extract-zip
==> default: npm http 304 https://registry.npmjs.org/hasha
==> default: npm http GET https://registry.npmjs.org/is-stream
==> default: npm http GET https://registry.npmjs.org/pinkie-promise
==> default: npm http GET https://registry.npmjs.org/is-absolute
==> default: npm http GET https://registry.npmjs.org/isexe
==> default: npm http GET https://registry.npmjs.org/concat-stream/1.5.0
==> default: npm http GET https://registry.npmjs.org/debug/0.7.4
==> default: npm http GET https://registry.npmjs.org/yauzl/2.4.1
==> default: npm http GET https://registry.npmjs.org/mkdirp/0.5.0
==> default: npm http GET https://registry.npmjs.org/throttleit
==> default: npm http 304 https://registry.npmjs.org/pinkie-promise
==> default: npm http GET https://registry.npmjs.org/jsonfile
==> default: npm http GET https://registry.npmjs.org/graceful-fs
==> default: npm http GET https://registry.npmjs.org/klaw
==> default: npm http GET https://registry.npmjs.org/path-is-absolute
==> default: npm http GET https://registry.npmjs.org/rimraf
==> default: npm http GET https://registry.npmjs.org/bl
==> default: npm http GET https://registry.npmjs.org/caseless
==> default: npm http GET https://registry.npmjs.org/forever-agent
==> default: npm http GET https://registry.npmjs.org/extend
==> default: npm http GET https://registry.npmjs.org/form-data
==> default: npm http GET https://registry.npmjs.org/json-stringify-safe
==> default: npm http GET https://registry.npmjs.org/mime-types
==> default: npm http GET https://registry.npmjs.org/node-uuid
==> default: npm http GET https://registry.npmjs.org/qs
==> default: npm http GET https://registry.npmjs.org/tunnel-agent
==> default: npm http GET https://registry.npmjs.org/tough-cookie
==> default: npm http GET https://registry.npmjs.org/http-signature
==> default: npm http GET https://registry.npmjs.org/oauth-sign
==> default: npm http GET https://registry.npmjs.org/hawk
==> default: npm http GET https://registry.npmjs.org/aws-sign2
==> default: npm http GET https://registry.npmjs.org/combined-stream
==> default: npm http GET https://registry.npmjs.org/stringstream
==> default: npm http GET https://registry.npmjs.org/isstream
==> default: npm http GET https://registry.npmjs.org/is-typedarray
==> default: npm http GET https://registry.npmjs.org/har-validator
==> default: npm http 304 https://registry.npmjs.org/is-stream
==> default: npm http GET https://registry.npmjs.org/pinkie
==> default: npm http 304 https://registry.npmjs.org/is-absolute
==> default: npm http 304 https://registry.npmjs.org/debug/0.7.4
==> default: npm http 304 https://registry.npmjs.org/concat-stream/1.5.0
==> default: npm http 304 https://registry.npmjs.org/isexe
==> default: npm http GET https://registry.npmjs.org/is-relative
==> default: npm http 304 https://registry.npmjs.org/graceful-fs
==> default: npm http 304 https://registry.npmjs.org/klaw
==> default: npm http 304 https://registry.npmjs.org/path-is-absolute
==> default: npm http 304 https://registry.npmjs.org/rimraf
==> default: npm http 304 https://registry.npmjs.org/bl
==> default: npm http 304 https://registry.npmjs.org/caseless
==> default: npm http 304 https://registry.npmjs.org/yauzl/2.4.1
==> default: npm http 304 https://registry.npmjs.org/forever-agent
==> default: npm http 304 https://registry.npmjs.org/extend
==> default: npm http 304 https://registry.npmjs.org/form-data
==> default: npm http 304 https://registry.npmjs.org/mkdirp/0.5.0
==> default: npm http 304 https://registry.npmjs.org/json-stringify-safe
==> default: npm http 200 https://registry.npmjs.org/node-uuid
==> default: npm http GET https://registry.npmjs.org/readable-stream
==> default: npm http GET https://registry.npmjs.org/inherits
==> default: npm http GET https://registry.npmjs.org/typedarray
==> default: npm http 304 https://registry.npmjs.org/tunnel-agent
==> default: npm http 200 https://registry.npmjs.org/mime-types
==> default: npm http 200 https://registry.npmjs.org/qs
==> default: npm http 304 https://registry.npmjs.org/http-signature
==> default: npm http GET https://registry.npmjs.org/fd-slicer
==> default: npm http 304 https://registry.npmjs.org/throttleit
==> default: npm http GET https://registry.npmjs.org/minimist/0.0.8
==> default: npm http 200 https://registry.npmjs.org/oauth-sign
==> default: npm http 304 https://registry.npmjs.org/jsonfile
==> default: npm http 304 https://registry.npmjs.org/hawk
==> default: npm http 304 https://registry.npmjs.org/aws-sign2
==> default: npm http 304 https://registry.npmjs.org/stringstream
==> default: npm http 200 https://registry.npmjs.org/combined-stream
==> default: npm http 304 https://registry.npmjs.org/isstream
==> default: npm http 304 https://registry.npmjs.org/is-typedarray
==> default: npm http GET https://registry.npmjs.org/glob
==> default: npm http 304 https://registry.npmjs.org/pinkie
==> default: npm http 304 https://registry.npmjs.org/readable-stream
==> default: npm http 304 https://registry.npmjs.org/har-validator
==> default: npm http 304 https://registry.npmjs.org/inherits
==> default: npm http 304 https://registry.npmjs.org/minimist/0.0.8
==> default: npm http 304 https://registry.npmjs.org/typedarray
==> default: npm http 304 https://registry.npmjs.org/glob
==> default: npm http 200 https://registry.npmjs.org/tough-cookie
==> default: npm http 304 https://registry.npmjs.org/fd-slicer
==> default: npm WARN engine hawk@3.1.3: wanted: {"node":">=0.10.32"} (current: {"node":"v0.10.25","npm":"1.3.10"})
==> default: npm http 304 https://registry.npmjs.org/is-relative
==> default: npm http GET https://registry.npmjs.org/process-nextick-args
==> default: npm http GET https://registry.npmjs.org/string_decoder
==> default: npm http GET https://registry.npmjs.org/util-deprecate
==> default: npm http GET https://registry.npmjs.org/core-util-is
==> default: npm http GET https://registry.npmjs.org/isarray
==> default: npm http GET https://registry.npmjs.org/delayed-stream
==> default: npm http GET https://registry.npmjs.org/mime-db
==> default: npm http 304 https://registry.npmjs.org/isarray
==> default: npm http 304 https://registry.npmjs.org/string_decoder
==> default: npm http 304 https://registry.npmjs.org/core-util-is
==> default: npm http 304 https://registry.npmjs.org/util-deprecate
==> default: npm http 304 https://registry.npmjs.org/process-nextick-args
==> default: npm http 200 https://registry.npmjs.org/mime-db
==> default: npm http GET https://registry.npmjs.org/pend
==> default: npm http GET https://registry.npmjs.org/minimatch
==> default: npm http GET https://registry.npmjs.org/once
==> default: npm http GET https://registry.npmjs.org/inflight
==> default: npm http GET https://registry.npmjs.org/async
==> default: npm http 304 https://registry.npmjs.org/delayed-stream
==> default: npm http GET https://registry.npmjs.org/assert-plus
==> default: npm http GET https://registry.npmjs.org/jsprim
==> default: npm http GET https://registry.npmjs.org/sshpk
==> default: npm http 304 https://registry.npmjs.org/async
==> default: npm http 304 https://registry.npmjs.org/minimatch
==> default: npm http 304 https://registry.npmjs.org/inflight
==> default: npm http 304 https://registry.npmjs.org/once
==> default: npm http 304 https://registry.npmjs.org/jsprim
==> default: npm http GET https://registry.npmjs.org/chalk
==> default: npm http GET https://registry.npmjs.org/is-my-json-valid
==> default: npm http GET https://registry.npmjs.org/commander
==> default: npm http GET https://registry.npmjs.org/wrappy
==> default: npm http GET https://registry.npmjs.org/brace-expansion
==> default: npm http 304 https://registry.npmjs.org/pend
==> default: npm http 304 https://registry.npmjs.org/is-my-json-valid
==> default: npm http 304 https://registry.npmjs.org/commander
==> default: npm http 304 https://registry.npmjs.org/assert-plus
==> default: npm http 304 https://registry.npmjs.org/wrappy
==> default: npm http 200 https://registry.npmjs.org/chalk
==> default: npm http 304 https://registry.npmjs.org/sshpk
==> default: npm http 304 https://registry.npmjs.org/brace-expansion
==> default: npm http GET https://registry.npmjs.org/has-ansi
==> default: npm http GET https://registry.npmjs.org/strip-ansi
==> default: npm http GET https://registry.npmjs.org/supports-color
==> default: npm http GET https://registry.npmjs.org/ansi-styles
==> default: npm http GET https://registry.npmjs.org/escape-string-regexp
==> default: npm http GET https://registry.npmjs.org/balanced-match
==> default: npm http GET https://registry.npmjs.org/concat-map/0.0.1
==> default: npm http GET https://registry.npmjs.org/verror/1.3.6
==> default: npm http GET https://registry.npmjs.org/extsprintf/1.0.2
==> default: npm http GET https://registry.npmjs.org/json-schema/0.2.2
==> default: npm http GET https://registry.npmjs.org/graceful-readlink
==> default: npm http GET https://registry.npmjs.org/hoek
==> default: npm http GET https://registry.npmjs.org/boom
==> default: npm http GET https://registry.npmjs.org/cryptiles
==> default: npm http GET https://registry.npmjs.org/sntp
==> default: npm http GET https://registry.npmjs.org/asn1
==> default: npm http GET https://registry.npmjs.org/dashdash
==> default: npm http GET https://registry.npmjs.org/getpass
==> default: npm http GET https://registry.npmjs.org/jsbn
==> default: npm http GET https://registry.npmjs.org/tweetnacl
==> default: npm http GET https://registry.npmjs.org/jodid25519
==> default: npm http GET https://registry.npmjs.org/ecc-jsbn
==> default: npm http GET https://registry.npmjs.org/generate-object-property
==> default: npm http GET https://registry.npmjs.org/jsonpointer/2.0.0
==> default: npm http GET https://registry.npmjs.org/xtend
==> default: npm http GET https://registry.npmjs.org/generate-function
==> default: npm http 304 https://registry.npmjs.org/has-ansi
==> default: npm http 304 https://registry.npmjs.org/balanced-match
==> default: npm http 200 https://registry.npmjs.org/strip-ansi
==> default: npm http 304 https://registry.npmjs.org/escape-string-regexp
==> default: npm http 304 https://registry.npmjs.org/concat-map/0.0.1
==> default: npm http 304 https://registry.npmjs.org/extsprintf/1.0.2
==> default: npm http 304 https://registry.npmjs.org/graceful-readlink
==> default: npm http 200 https://registry.npmjs.org/ansi-styles
==> default: npm http 304 https://registry.npmjs.org/hoek
==> default: npm http 304 https://registry.npmjs.org/verror/1.3.6
==> default: npm http 304 https://registry.npmjs.org/cryptiles
==> default: npm http 304 https://registry.npmjs.org/json-schema/0.2.2
==> default: npm http 304 https://registry.npmjs.org/sntp
==> default: npm http 304 https://registry.npmjs.org/supports-color
==> default: npm http 304 https://registry.npmjs.org/asn1
==> default: npm http GET https://registry.npmjs.org/ansi-regex
==> default: npm http 304 https://registry.npmjs.org/jsbn
==> default: npm http 304 https://registry.npmjs.org/getpass
==> default: npm http 304 https://registry.npmjs.org/jodid25519
==> default: npm http 304 https://registry.npmjs.org/boom
==> default: npm WARN engine hoek@2.16.3: wanted: {"node":">=0.10.40"} (current: {"node":"v0.10.25","npm":"1.3.10"})
==> default: npm WARN engine cryptiles@2.0.5: wanted: {"node":">=0.10.40"} (current: {"node":"v0.10.25","npm":"1.3.10"})
==> default: npm WARN engine boom@2.10.1: wanted: {"node":">=0.10.40"} (current: {"node":"v0.10.25","npm":"1.3.10"})
==> default: npm http 304 https://registry.npmjs.org/jsonpointer/2.0.0
==> default: npm http 304 https://registry.npmjs.org/ecc-jsbn
==> default: npm http 304 https://registry.npmjs.org/generate-function
==> default: npm http 304 https://registry.npmjs.org/dashdash
==> default: npm http 304 https://registry.npmjs.org/tweetnacl
==> default: npm http 200 https://registry.npmjs.org/ansi-regex
==> default: npm http 304 https://registry.npmjs.org/generate-object-property
==> default: npm http 304 https://registry.npmjs.org/xtend
==> default: npm http GET https://registry.npmjs.org/is-property
==> default: npm http 304 https://registry.npmjs.org/is-property
==> default: Error checking path, continuing { [Error: Cannot find module '/usr/local/lib/node_modules/phantomjs/lib/location'] code: 'MODULE_NOT_FOUND' }
==> default: Error extracting archive
==> default: Phantom installation failed { [Error: Command failed:
==> default: bzip2: Compressed file ends unexpectedly;
==> default:    perhaps it is corrupted?  *Possible* reason follows.
==> default: bzip2: Inappropriate ioctl for device
==> default:    Input file = (stdin), output file = (stdout)
==> default:
==> default: It is possible that the compressed file(s) have become corrupted.
==> default: You can use the -tvv option to test integrity of such files.
==> default:
==> default: You can use the `bzip2recover' program to attempt to recover
==> default: data from undamaged sections of corrupted files.
==> default:
==> default: tar: Unexpected EOF in archive
==> default: tar: Unexpected EOF in archive
==> default: tar: Error is not recoverable: exiting now
==> default: ] killed: false, code: 2, signal: null } Error: Command failed:
==> default: bzip2: Compressed file ends unexpectedly;
==> default:    perhaps it is corrupted?  *Possible* reason follows.
==> default: bzip2: Inappropriate ioctl for device
==> default:    Input file = (stdin), output file = (stdout)
==> default:
==> default: It is possible that the compressed file(s) have become corrupted.
==> default: You can use the -tvv option to test integrity of such files.
==> default:
==> default: You can use the `bzip2recover' program to attempt to recover
==> default: data from undamaged sections of corrupted files.
==> default:
==> default: tar: Unexpected EOF in archive
==> default: tar: Unexpected EOF in archive
==> default: tar: Error is not recoverable: exiting now
==> default:
==> default:     at ChildProcess.exithandler (child_process.js:637:15)
==> default:     at ChildProcess.EventEmitter.emit (events.js:98:17)
==> default:     at maybeClose (child_process.js:743:16)
==> default:     at Process.ChildProcess._handle.onexit (child_process.js:810:5)
==> default: npm WARN This failure might be due to the use of legacy binary "node"
==> default: npm WARN For further explanations, please read
==> default: /usr/share/doc/nodejs/README.Debian
==> default:
==> default: npm ERR! weird error 1
==> default: npm ERR! not ok code 0
==> default: stdout: /usr/local/bin/phantomjs -> /usr/local/lib/node_modules/phantomjs/bin/phantomjs
==> default:
==> default: > phantomjs@2.1.7 install /usr/local/lib/node_modules/phantomjs
==> default: > node install.js
==> default:
==> default: Considering PhantomJS found at /usr/local/bin/phantomjs
==> default: Looks like an `npm install -g`
==> default: Downloading https://github.com/Medium/phantomjs/releases/download/v2.1.1//phantomjs-2.1.1-linux-x86_64.tar.bz2
==> default: Saving to /usr/local/lib/node_modules/phantomjs/phantomjs/phantomjs-2.1.1-linux-x86_64.tar.bz2
==> default: Receiving...
==> default:
==> default: Received 5218K total.
==> default: Extracting tar contents (via spawned process)
==> default:
==> default: msg: npm http GET https://registry.npmjs.org/phantomjs
==> default: npm http 304 https:********@2.1.7: Package renamed to phantomjs-prebuilt. Please update 'phantomjs' package references to 'phantomjs-prebuilt'
==> default: npm http GET https://registry.npmjs.org/extract-zip
==> default: npm http GET https://registry.npmjs.org/fs-extra
==> default: npm http GET https://registry.npmjs.org/hasha
==> default: npm http GET https://registry.npmjs.org/progress
==> default: npm http GET https://registry.npmjs.org/kew
==> default: npm http GET https://registry.npmjs.org/request
==> default: npm http GET https://registry.npmjs.org/request-progress
==> default: npm http GET https://registry.npmjs.org/which
==> default: npm http 304 https://registry.npmjs.org/fs-extra
==> default: npm http 304 https://registry.npmjs.org/progress
==> default: npm http 304 https://registry.npmjs.org/kew
==> default: npm http 304 https://registry.npmjs.org/request
==> default: npm http 304 https://registry.npmjs.org/request-progress
==> default: npm http 304 https://registry.npmjs.org/which
==> default: npm http 304 https://registry.npmjs.org/extract-zip
==> default: npm http 304 https://registry.npmjs.org/hasha
==> default: npm http GET https://registry.npmjs.org/is-stream
==> default: npm http GET https://registry.npmjs.org/pinkie-promise
==> default: npm http GET https://registry.npmjs.org/is-absolute
==> default: npm http GET https://registry.npmjs.org/isexe
==> default: npm http GET https://registry.npmjs.org/concat-stream/1.5.0
==> default: npm http GET https://registry.npmjs.org/debug/0.7.4
==> default: npm http GET https://registry.npmjs.org/yauzl/2.4.1
==> default: npm http GET https://registry.npmjs.org/mkdirp/0.5.0
==> default: npm http GET https://registry.npmjs.org/throttleit
==> default: npm http 304 https://registry.npmjs.org/pinkie-promise
==> default: npm http GET https://registry.npmjs.org/jsonfile
==> default: npm http GET https://registry.npmjs.org/graceful-fs
==> default: npm http GET https://registry.npmjs.org/klaw
==> default: npm http GET https://registry.npmjs.org/path-is-absolute
==> default: npm http GET https://registry.npmjs.org/rimraf
==> default: npm http GET https://registry.npmjs.org/bl
==> default: npm http GET https://registry.npmjs.org/caseless
==> default: npm http GET https://registry.npmjs.org/forever-agent
==> default: npm http GET https://registry.npmjs.org/extend
==> default: npm http GET https://registry.npmjs.org/form-data
==> default: npm http GET https://registry.npmjs.org/json-stringify-safe
==> default: npm http GET https://registry.npmjs.org/mime-types
==> default: npm http GET https://registry.npmjs.org/node-uuid
==> default: npm http GET https://registry.npmjs.org/qs
==> default: npm http GET https://registry.npmjs.org/tunnel-agent
==> default: npm http GET https://registry.npmjs.org/tough-cookie
==> default: npm http GET https://registry.npmjs.org/http-signature
==> default: npm http GET https://registry.npmjs.org/oauth-sign
==> default: npm http GET https://registry.npmjs.org/hawk
==> default: npm http GET https://registry.npmjs.org/aws-sign2
==> default: npm http GET https://registry.npmjs.org/combined-stream
==> default: npm http GET https://registry.npmjs.org/stringstream
==> default: npm http GET https://registry.npmjs.org/isstream
==> default: npm http GET https://registry.npmjs.org/is-typedarray
==> default: npm http GET https://registry.npmjs.org/har-validator
==> default: npm http 304 https://registry.npmjs.org/is-stream
==> default: npm http GET https://registry.npmjs.org/pinkie
==> default: npm http 304 https://registry.npmjs.org/is-absolute
==> default: npm http 304 https://registry.npmjs.org/debug/0.7.4
==> default: npm http 304 https://registry.npmjs.org/concat-stream/1.5.0
==> default: npm http 304 https://registry.npmjs.org/isexe
==> default: npm http GET https://registry.npmjs.org/is-relative
==> default: npm http 304 https://registry.npmjs.org/graceful-fs
==> default: npm http 304 https://registry.npmjs.org/klaw
==> default: npm http 304 https://registry.npmjs.org/path-is-absolute
==> default: npm http 304 https://registry.npmjs.org/rimraf
==> default: npm http 304 https://registry.npmjs.org/bl
==> default: npm http 304 https://registry.npmjs.org/caseless
==> default: npm http 304 https://registry.npmjs.org/yauzl/2.4.1
==> default: npm http 304 https://registry.npmjs.org/forever-agent
==> default: npm http 304 https://registry.npmjs.org/extend
==> default: npm http 304 https://registry.npmjs.org/form-data
==> default: npm http 304 https://registry.npmjs.org/mkdirp/0.5.0
==> default: npm http 304 https://registry.npmjs.org/json-stringify-safe
==> default: npm http 200 https://registry.npmjs.org/node-uuid
==> default: npm http GET https://registry.npmjs.org/readable-stream
==> default: npm http GET https://registry.npmjs.org/inherits
==> default: npm http GET https://registry.npmjs.org/typedarray
==> default: npm http 304 https://registry.npmjs.org/tunnel-agent
==> default: npm http 200 https://registry.npmjs.org/mime-types
==> default: npm http 200 https://registry.npmjs.org/qs
==> default: npm http 304 https://registry.npmjs.org/http-signature
==> default: npm http GET https://registry.npmjs.org/fd-slicer
==> default: npm http 304 https://registry.npmjs.org/throttleit
==> default: npm http GET https://registry.npmjs.org/minimist/0.0.8
==> default: npm http 200 https://registry.npmjs.org/oauth-sign
==> default: npm http 304 https://registry.npmjs.org/jsonfile
==> default: npm http 304 https://registry.npmjs.org/hawk
==> default: npm http 304 https://registry.npmjs.org/aws-sign2
==> default: npm http 304 https://registry.npmjs.org/stringstream
==> default: npm http 200 https://registry.npmjs.org/combined-stream
==> default: npm http 304 https://registry.npmjs.org/isstream
==> default: npm http 304 https://registry.npmjs.org/is-typedarray
==> default: npm http GET https://registry.npmjs.org/glob
==> default: npm http 304 https://registry.npmjs.org/pinkie
==> default: npm http 304 https://registry.npmjs.org/readable-stream
==> default: npm http 304 https://registry.npmjs.org/har-validator
==> default: npm http 304 https://registry.npmjs.org/inherits
==> default: npm http 304 https://registry.npmjs.org/minimist/0.0.8
==> default: npm http 304 https://registry.npmjs.org/typedarray
==> default: npm http 304 https://registry.npmjs.org/glob
==> default: npm http 200 https://registry.npmjs.org/tough-cookie
==> default: npm http 304 https:********@3.1.3: wanted: {"node":">=0.10.32"} (current: {"node":"v0.10.25","npm":"1.3.10"})
==> default: npm http 304 https://registry.npmjs.org/is-relative
==> default: npm http GET https://registry.npmjs.org/process-nextick-args
==> default: npm http GET https://registry.npmjs.org/string_decoder
==> default: npm http GET https://registry.npmjs.org/util-deprecate
==> default: npm http GET https://registry.npmjs.org/core-util-is
==> default: npm http GET https://registry.npmjs.org/isarray
==> default: npm http GET https://registry.npmjs.org/delayed-stream
==> default: npm http GET https://registry.npmjs.org/mime-db
==> default: npm http 304 https://registry.npmjs.org/isarray
==> default: npm http 304 https://registry.npmjs.org/string_decoder
==> default: npm http 304 https://registry.npmjs.org/core-util-is
==> default: npm http 304 https://registry.npmjs.org/util-deprecate
==> default: npm http 304 https://registry.npmjs.org/process-nextick-args
==> default: npm http 200 https://registry.npmjs.org/mime-db
==> default: npm http GET https://registry.npmjs.org/pend
==> default: npm http GET https://registry.npmjs.org/minimatch
==> default: npm http GET https://registry.npmjs.org/once
==> default: npm http GET https://registry.npmjs.org/inflight
==> default: npm http GET https://registry.npmjs.org/async
==> default: npm http 304 https://registry.npmjs.org/delayed-stream
==> default: npm http GET https://registry.npmjs.org/assert-plus
==> default: npm http GET https://registry.npmjs.org/jsprim
==> default: npm http GET https://registry.npmjs.org/sshpk
==> default: npm http 304 https://registry.npmjs.org/async
==> default: npm http 304 https://registry.npmjs.org/minimatch
==> default: npm http 304 https://registry.npmjs.org/inflight
==> default: npm http 304 https://registry.npmjs.org/once
==> default: npm http 304 https://registry.npmjs.org/jsprim
==> default: npm http GET https://registry.npmjs.org/chalk
==> default: npm http GET https://registry.npmjs.org/is-my-json-valid
==> default: npm http GET https://registry.npmjs.org/commander
==> default: npm http GET https://registry.npmjs.org/wrappy
==> default: npm http GET https://registry.npmjs.org/brace-expansion
==> default: npm http 304 https://registry.npmjs.org/pend
==> default: npm http 304 https://registry.npmjs.org/is-my-json-valid
==> default: npm http 304 https://registry.npmjs.org/commander
==> default: npm http 304 https://registry.npmjs.org/assert-plus
==> default: npm http 304 https://registry.npmjs.org/wrappy
==> default: npm http 200 https://registry.npmjs.org/chalk
==> default: npm http 304 https://registry.npmjs.org/sshpk
==> default: npm http 304 https://registry.npmjs.org/brace-expansion
==> default: npm http GET https://registry.npmjs.org/has-ansi
==> default: npm http GET https://registry.npmjs.org/strip-ansi
==> default: npm http GET https://registry.npmjs.org/supports-color
==> default: npm http GET https://registry.npmjs.org/ansi-styles
==> default: npm http GET https://registry.npmjs.org/escape-string-regexp
==> default: npm http GET https://registry.npmjs.org/balanced-match
==> default: npm http GET https://registry.npmjs.org/concat-map/0.0.1
==> default: npm http GET https://registry.npmjs.org/verror/1.3.6
==> default: npm http GET https://registry.npmjs.org/extsprintf/1.0.2
==> default: npm http GET https://registry.npmjs.org/json-schema/0.2.2
==> default: npm http GET https://registry.npmjs.org/graceful-readlink
==> default: npm http GET https://registry.npmjs.org/hoek
==> default: npm http GET https://registry.npmjs.org/boom
==> default: npm http GET https://registry.npmjs.org/cryptiles
==> default: npm http GET https://registry.npmjs.org/sntp
==> default: npm http GET https://registry.npmjs.org/asn1
==> default: npm http GET https://registry.npmjs.org/dashdash
==> default: npm http GET https://registry.npmjs.org/getpass
==> default: npm http GET https://registry.npmjs.org/jsbn
==> default: npm http GET https://registry.npmjs.org/tweetnacl
==> default: npm http GET https://registry.npmjs.org/jodid25519
==> default: npm http GET https://registry.npmjs.org/ecc-jsbn
==> default: npm http GET https://registry.npmjs.org/generate-object-property
==> default: npm http GET https://registry.npmjs.org/jsonpointer/2.0.0
==> default: npm http GET https://registry.npmjs.org/xtend
==> default: npm http GET https://registry.npmjs.org/generate-function
==> default: npm http 304 https://registry.npmjs.org/has-ansi
==> default: npm http 304 https://registry.npmjs.org/balanced-match
==> default: npm http 200 https://registry.npmjs.org/strip-ansi
==> default: npm http 304 https://registry.npmjs.org/escape-string-regexp
==> default: npm http 304 https://registry.npmjs.org/concat-map/0.0.1
==> default: npm http 304 https://registry.npmjs.org/extsprintf/1.0.2
==> default: npm http 304 https://registry.npmjs.org/graceful-readlink
==> default: npm http 200 https://registry.npmjs.org/ansi-styles
==> default: npm http 304 https://registry.npmjs.org/hoek
==> default: npm http 304 https://registry.npmjs.org/verror/1.3.6
==> default: npm http 304 https://registry.npmjs.org/cryptiles
==> default: npm http 304 https://registry.npmjs.org/json-schema/0.2.2
==> default: npm http 304 https://registry.npmjs.org/sntp
==> default: npm http 304 https://registry.npmjs.org/supports-color
==> default: npm http 304 https://registry.npmjs.org/asn1
==> default: npm http GET https://registry.npmjs.org/ansi-regex
==> default: npm http 304 https://registry.npmjs.org/jsbn
==> default: npm http 304 https://registry.npmjs.org/getpass
==> default: npm http 304 https://registry.npmjs.org/jodid25519
==> default: npm http 304 https://registry.npmjs.org/boom
==> default: npm WARN engine hoek@2.16.3:********@2.10.1: wanted: {"node":">=0.10.40"} (current: {"node":"v0.10.25","npm":"1.3.10"})
==> default: npm http 304 https://registry.npmjs.org/jsonpointer/2.0.0
==> default: npm http 304 https://registry.npmjs.org/ecc-jsbn
==> default: npm http 304 https://registry.npmjs.org/generate-function
==> default: npm http 304 https://registry.npmjs.org/dashdash
==> default: npm http 304 https://registry.npmjs.org/tweetnacl
==> default: npm http 200 https://registry.npmjs.org/ansi-regex
==> default: npm http 304 https://registry.npmjs.org/generate-object-property
==> default: npm http 304 https://registry.npmjs.org/xtend
==> default: npm http GET https://registry.npmjs.org/is-property
==> default: npm http 304 https://registry.npmjs.org/is-property
==> default: Error checking path, continuing { [Error: Cannot find module '/usr/local/lib/node_modules/phantomjs/lib/location'] code: 'MODULE_NOT_FOUND' }
==> default: Error extracting archive
==> default: Phantom installation failed { [Error: Command failed:
==> default: bzip2: Compressed file ends unexpectedly;
==> default:    perhaps it is corrupted?  *Possible* reason follows.
==> default: bzip2: Inappropriate ioctl for device
==> default:    Input file = (stdin), output file = (stdout)
==> default:
==> default: It is possible that the compressed file(s) have become corrupted.
==> default: You can use the -tvv option to test integrity of such files.
==> default:
==> default: You can use the `bzip2recover' program to attempt to recover
==> default: data from undamaged sections of corrupted files.
==> default:
==> default: tar: Unexpected EOF in archive
==> default: tar: Unexpected EOF in archive
==> default: tar: Error is not recoverable: exiting now
==> default: ] killed: false, code: 2, signal: null } Error: Command failed:
==> default: bzip2: Compressed file ends unexpectedly;
==> default:    perhaps it is corrupted?  *Possible* reason follows.
==> default: bzip2: Inappropriate ioctl for device
==> default:    Input file = (stdin), output file = (stdout)
==> default:
==> default: It is possible that the compressed file(s) have become corrupted.
==> default: You can use the -tvv option to test integrity of such files.
==> default:
==> default: You can use the `bzip2recover' program to attempt to recover
==> default: data from undamaged sections of corrupted files.
==> default:
==> default: tar: Unexpected EOF in archive
==> default: tar: Unexpected EOF in archive
==> default: tar: Error is not recoverable: exiting now
==> default:
==> default:     at ChildProcess.exithandler (child_process.js:637:15)
==> default:     at ChildProcess.EventEmitter.emit (events.js:98:17)
==> default:     at maybeClose (child_process.js:743:16)
==> default:     at Process.ChildProcess._handle.onexit (child_process.js:810:5)
==> default: npm WARN This failure might be due to the use of legacy binary "node"
==> default: npm WARN For further explanations, please read
==> default: /usr/share/doc/nodejs/README.Debian
==> default:
==> default: npm ERR! weird error 1
==> default: npm ERR! not ok code 0
==> default: ok: [localhost] => (item=typescript)
==> default: ok: [localhost] => (item=jshint)
==> default: ok: [localhost] => (item=jslint)
==> default:
==> default: FATAL: all hosts have already failed -- aborting
==> default:
==> default: PLAY RECAP ********************************************************************
==> default:            to retry, use: --limit @/root/playbook.retry
==> default:
==> default: localhost                  : ok=28   changed=2    unreachable=0    failed=1
The SSH command responded with a non-zero exit status. Vagrant
assumes that this means the command failed. The output for this command
should be in the log above. Please read the output to determine what
went wrong.
```

**Comment by mblaschke** [comment]
hmm.. seems to be an upstream issue
Maybe we should remove this stuff in the next release, not everyone needs phantomjs.

Workaround: remove task in `provision/ansible/roles/common/tasks/packages/npm.yml`

**Comment by mblaschke** [comment]
New customized package installation is available in `ubuntu-16.04` which fixes these issues.
